### PR TITLE
integrate Jacoco to project

### PR DESCRIPTION
**Pull request by AndriyPavlyuk**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,6 +12,11 @@ repositories {
     mavenCentral()
 }
 
+jacoco {
+    toolVersion = "0.8.12"
```
Comment (DimaGalchenko) [inline build.gradle:16]: Move this to version.gradle file, where we store all dependency versions

**Review by DimaGalchenko — CHANGES_REQUESTED** [review]
LGTM, one minor:

File: build.gradle (hunk 1 above)
Comment (AndriyPavlyuk) [reply build.gradle:16]: Thank you!

Comment (AndriyPavlyuk) [reply build.gradle:16]: Moved

**Pull request closed without merging**